.pre-commit-config.yaml

```diff
@@ -60,3 +60,20 @@ repos:
     hooks:
       - id: yamllint
         files: ^tmt/schemas/.*\.yaml
+
+  # Yet another static analysis, but these hooks "understand" Python syntax and semantics, and
```

Comment:
I was reading this comment and I am not sure if it is very helpful :( 

I would just mention some additional hardening on # noqa and # ignore usage?

Reply:
Yeah, it's misleading. Rewritten & rebased.